pages/templates/pages/election_tracking_page.html

```diff
@@ -95,28 +95,45 @@
                                           {% with candidate_race.candidate as candidate %}
 
                                               <li class="card clearfix">
-                                                  <div class="card__photo" style="background-image: url({{ candidate.photo.url }})"></div>
-                                                    <div class="card__content">
-                                                      <div class="candidate__info">
-                                                        <span class="card__item bold bigger name wins" sort-by="wins">{{ candidate.name }}</span>
-                                                        <span class="card__item smaller"><span class="state">{{ candidate.state }}</span> {{ candidate.district }}</span>
-                                                        <span class="card__item bottom">{{ candidate.office }}</span>
-                                                    </div>
 
-                                                    {% comment %}
-                                                      <div class="card__item card__item--share">
-                                                        <a href="#" class="social-icon facebook-icon facebook-bg">Share on Facebook</a>
-                                                        <a href="https://twitter.com/intent/tweet?text={% spaceless %}
-                                                              {% if candidate_race.candidate_votes == 0 and candidate_race.opponent_votes == 0 %}
-                                                                  Polls for {{ candidate.name }} aren't in yet. Track this race and other progressive's results nationwide LIVE.
-                                                              {% elif candidate_race.candidate_votes > candidate_race.opponent_votes %}
-                                                                  {{ candidate.name }} is leading {{ candidate_race.candidate_votes_percentage | urlencode }} to {{ candidate_race.opponent_votes_percentage | urlencode }}. Track this race and other progressive's results nationwide LIVE.
-                                                              {% else %}
-                                                                  {{ candidate.name }} is trailing {{ candidate_race.opponent_votes_percentage | urlencode }} to {{ candidate_race.candidate_votes_percentage | urlencode }}. Track this race and other progressive's results nationwide LIVE.
-                                                              {% endif %}{% endspaceless %}&url=https://ourrevolution.com/election-2016" class="social-icon twitter-icon twitter-bg">Tweet</a>
-                                                      </div>
-                                                    {% endcomment %}
+                                                {% if candidate_race.candidate_endorsement_page %}
```

Comment:
There's no visual indication to the user that these are links. I recommend we link the entire card and add hover effect seen on candidate page, or perhaps make text blue and add a `glyphicon glyphicon-chevron-right` as a visual indicator.

Reply:
I think right now they already look like links, except you can't click them. I think actually making it a link with no other design changes will be intuitive for users. I have clicked on these many times thinking it was a link and been confused that it wasn't. On the candidates page all the cards and photos are clickable so we have already set expectation that these cards should be clickable.

Reply:
Afk atm but if I recall correctly only the photos were linked here, correct? At minimum should be entire card similar to candidates page so we're consistent and so it's intuitive for user. We don't necessarily need same hover functionality at the moment but should create issue to make consistent cross-site in future.

Reply:
It's not just photo link, its photo plus name/office/text content - but not results. Similar to candidates page.

I thought about linking results too but it seemed counterintuitive - maybe it should have separate field for linking to actual results article rather than candidate page.

Also the other problem that came up is the social media icons are also links, so if they are enabled we can't have links inside of links.

Reply:
Just reviewed and still only linking photo - are you sure? Don't need to worry about social media links for now.